deployment/benchmarks/3scale-benchmark.yaml.j2

```diff
@@ -0,0 +1,52 @@
+name: 3scale-benchmark
+agents:
+{% for agent in groups[hyperfoil_agent_group] %}
+  {{ agent }}: {{ hostvars[agent]['ansible_host'] }}:{{ hyperfoil_agent_port }}
+{% endfor %}
+http:
+{% for host in groups[injector_target_group] %}

```

Comment:
Is this section really needed?

The `host` field is used in `httpRequest` task directly. 

If the answer is, indeed, not needed, then, another question is: Is `3scale_inventory_plugin.py` needed?

Reply:
> Is this section really needed?
 It is. See [HTTP](https://hyperfoil.io/userguide/benchmark/http.html).

Reply:
And what happens if host defined in the `httpRequest` section does not belong to the list of the `http` section?

Reply:
That should never happen.
 traffic generator source CSV file is used as the data source, the value used in both httpRequest and http section come from there.